Styling/README.md

```diff
@@ -1340,19 +1342,11 @@ For example:
 }
 ```
 
-#### Point Cloud Shader Styling
-
-**TODO : add note about GLSL implementations requires strict type comparisons among other things: https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/140**
-
-## File Extension
-
-TBA
-
-## MIME Type
-
-_TBA, [#60](https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/60)_
+> Implementation Note: Point cloud styling engines may often use a shader (GLSL) implementation, however some features of the expression language are not possible in pure a GLSL implementation. Features that must account for these inconsistencies include evaluation of `isNan` and `isFinite` (GLSL 2.0+ supports `isnan` and `isinf` for these functions respectively); the types `null` and `undefined`; strings, including accessing object properties (`color()['r']`) and batch table values; regular expressions; arrays of lengths other than 2, 3, or 4; mismatched type comparisons (`float` to `bool`); and handling of "index out of bounds" errors.
```

Comment:
Is it possible to have a bulleted list within a quoted section? That might be easier to read.